Bottom: 1d64c0ad0821d2fbf215518388675ce707d2e01a
Top:    6b0a5f7c4a9973e6f3e9ea683e49a256833cf9bb
Author: Ian Kent <raven@themaw.net>
Date:   2013-06-11 13:57:14 +0800

autofs-5.0.7 - add changlog entry for coverity fixes

A bunch of changes have been made based on a Covarity report.
Mostly I pust the changes into seperate patches so they can be kept simple.

Changes are:
- fix fcntl return check.
- fix spawn_umount() return check in mount_bind.c:lookup_init().
- fix check mkdir_path() in mount_bind.c:mount_mount().
- fix incorrect var name in test.


---

diff --git a/CHANGELOG b/CHANGELOG
index 48e9806..4ad1e43 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -49,6 +49,8 @@
 - fix master map bogus keywork match.
 - fix fix map entry duplicate offset detection.
 - probe each nfs version in turn for singleton mounts.
+- add changlog entry for coverity fixes.
+
 
 25/07/2012 autofs-5.0.7
 =======================
